plugin/code_actions.py

```diff
@@ -128,7 +131,6 @@ def _request_async(
                 cache_key, cache_collector = self._response_cache
                 if location_cache_key == cache_key:
                     sublime.set_timeout(lambda: actions_handler(cache_collector.get_actions()))
-                    return cache_collector
```

Comment:
This return should not have been removed as it changes the logic.

Reply:
I will make a PR tonight that will hopefully solve each of the comments above  :)